django/db/backends/sqlite3/creation.py

```diff
@@ -133,17 +133,18 @@ def setup_worker_connection(self, _worker_id):
         # reflected in django.db.connections. Otherwise new threads would
         # connect to the default database instead of the appropriate clone.
         start_method = multiprocessing.get_start_method()
-        if start_method == "fork":
+        alias = self.connection.alias
+        source_db_file = f"{alias}_{_worker_id}.sqlite3"
+        if start_method == "fork" or not os.path.isfile(source_db_file):
             # Update settings_dict in place.
             self.connection.settings_dict.update(settings_dict)
             self.connection.close()
```

Comment:
I was thinking we could clean this method up though because these lines are straight out of `super().setup_worker_connection()`

Reply:
It may be a good cleanup but I would have that in a separated PR if possible. Thanks!